src/vector/tasks.yaml

```diff
@@ -11,3 +11,9 @@ tasks:
             name: app.kubernetes.io/name=vector
             namespace: vector
             condition: Ready
+  - name: e2e-test
+    actions:
+      - description: "Run Vector integration tests"
+        cmd: |
+            npm ci && npx jest --testPathPattern vector* || echo "No tests found for vector, skipping..."
```

Comment:
Should the other stubbed out tests follow this pattern as well?

Reply:
We can add this to the other stubbed tests if we make the assumption they will be jest tests.

Reply:
Ah tracking, maybe we just stub this one out the same as the others for now since we don't have any vector-only tests anymore? Definitely a nit, we could leave this as-is too.